src/post_processor.cpp

```diff
@@ -24,6 +24,41 @@
 #define MIN_CONF_THRESH 0.0  // Minimum value a confidence threshold can be
 #define MAX_CONF_THRESH 1.0  // Maximum value a confidence threshold can be
 
+/**
+ * @brief Take the mean of two body parts
+ *
+ * Intended for use with two body parts that lie left and right of the spine
+ * (as an axis of symmetry). As the system is for use with the person in
+ * profile, these should overlap, but there may be slight differences.
+ *
+ * If both input `PostProcessing::Coordinate`s are
+ * `PostProcessing::Trustworthy` the arithmetic mean of the two is returned (x
+ * and y coordinates are averaged independently). If only one body part is
+ * `PostProcessing::Trustworthy` only that body part is returned, i.e., no
+ * averaging takes place. In the case where both are
+ * `PostProcessing::Untrustworthy`, the mean is returned, although this may
+ * not be particularly meaningful as indicated by the `Untrustworthy` flag.
+ *
+ * @param left a `Inference::Coordinate`
+ * @param right a `Inference::Coordinate`
+ * @return `PostProcessing::Coordinate` the "intelligent" mean
+ */
+PostProcessing::Coordinate intelligent_mean_body_parts(
+    PostProcessing::Coordinate left, PostProcessing::Coordinate right) {
+  if (left.status == PostProcessing::Trustworthy &&
+      right.status == PostProcessing::Untrustworthy) {
+    return left;
+  } else if (left.status == PostProcessing::Untrustworthy &&
+             right.status == PostProcessing::Trustworthy) {
+    return right;
+  } else {
+    // Both Trustworthy or Untrustworthy
+    float x = (left.x + right.x) / 2;
```

Comment:
Should we do any weighing based on accuracy. Like is one shoulder is trustworthy but 99% and the other is trustworthy but just 60%, should the 99% get more preference? Or is the threshold always high enough that they won't be more than a few percent difference?

Reply:
Discussed: we're going with 50-50 and not weighting, but could change in the future.